com.musala.database.web.spa.parser/WebContent/js/controller/ConnectionController.js

```diff
@@ -38,6 +38,7 @@ app.controller('ConnectionController', function($scope, $http, $rootScope) {
 				}, 1000);
 			}
 		},function(fail){
+			console.log("http fail");
```

Comment:
Please remove this unnecessary console log or either add such a logging message to the fail clauses of all controllers.